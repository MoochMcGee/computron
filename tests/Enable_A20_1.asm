[bits 16]

mov dx, 0x92
mov al, 2
out dx, al

call check_a20

db 0xf1

%include "check_a20.inc"
