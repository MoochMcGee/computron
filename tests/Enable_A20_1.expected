(  config) Loading tmp.bin at 0x00010000
(  screen) No keymap to load!
(    init) BusMouse present
(    init) FDC present
(    init) IDE present
(    init) Keyboard present
(    init) PIC present
(    init) PIC present
(    init) PIT present
(    init) VGA present
(    init) VomCtl present
(    init) CMOS present
(    init) PS2 present
(     cpu) 0xF1: Secret shutdown command received!
(    dump) AX: 0001
(    dump) BX: 0000
(    dump) CX: 0000
(    dump) DX: 0092
(    dump) BP: 0000
(    dump) SP: 0000
(    dump) SI: 0000
(    dump) DI: 0000
(    dump) CS: 1000
(    dump) DS: 0000
(    dump) ES: 0000
(    dump) SS: 0000
(    dump) FS: 0000
(    dump) GS: 0000
(    dump) CR0: 00000000
(    dump) A20: 1
(    dump) GDTR: {base=00000000, limit=0000}
(    dump) LDTR: {base=00000000, limit=0000}
(    dump) IDTR: {base=00000000, limit=0000}
(    dump) C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0
(    dump)   -  (BA 9200B002EE)
(    dump) 1000:00000009 F1             ??? 
